init/postrm

```diff
@@ -0,0 +1,10 @@
+### ONLY if purging, remove database and user
+if [ "$1" = purge ]; then
+	# remove haulage_db user and database
+	sudo mysql -e "DROP DATABASE IF EXISTS haulage_db;"
+	sudo mysql -e "DROP USER IF EXISTS haulage_db@localhost;"
+	sudo mysql -e "FLUSH PRIVILEGES;"
+	echo "Purged haulage_db user and database"
+fi
+
+echo "Uninstalled haulage"
```

Comment:
Do we need to make any systemd calls as part of postrm now that we're not using FPM's systemd integration?